config/webpack.dev.js

```diff
@@ -0,0 +1,81 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const webpack = require('webpack');
```

Comment:
When you have several repeating parts in both configs, consider using webpack-merge https://www.npmjs.com/package/webpack-merge